vueui/src/components/view/admin/extended-list.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div>
+    <table>
+      <ul v-for="listitem in listvar" :key="listitem.key">
```

Comment:
v-bind key is with totally invalid value

Reply:
yeah i forgot how to code for a second, happens to the best of us

and also me